src/ucp/core/ucp_am.c

```diff
@@ -425,6 +440,35 @@ ucs_status_t ucp_worker_set_am_recv_handler(ucp_worker_h worker,
     return status;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_am_copy_user_header_to_internal_buf(ucp_request_t *req)
```

Comment:
pls align naming

Reply:
ucp_am_copy_header_to_internal_buf?

Reply:
IMHO such functions are expected to be symmetic - put/get, copy_to/copy_from, in/out, etc